### ISSUE TITLE
Page size very large

### ISSUE THREAD

**Issue by snoopen** [maxvfischer/DIY-CNC-machine#4]
The whole page is very heavy at over 130mb. Additionally, the long single page format is difficult to navigate.

Some ideas to consider:

- Use mp4 instead of gif
- Move each step into its own page:
  - Host the guide on Github Pages and maybe lazy load the images
  - Sticking to github only, each section could be a folder with its own readme.md

**Comment by maxvfischer** [comment]
I totally agree. I started to write a script to split the readme into separate readme's, then add "<- Prev" and "Next ->" links at the bottom and top of the page for easier navigation. I'll jump into it when I some spare time.
